File: PersianCalendar/src/main/java/com/byagowi/persiancalendar/entities/IslamicCalendarEvent.java
package com.byagowi.persiancalendar.entities;

import com.byagowi.persiancalendar.calendar.IslamicDate;

public class IslamicCalendarEvent extends AbstractEvent<IslamicDate> {
    public IslamicCalendarEvent(IslamicDate date, String title, boolean holiday) {
        this.date = date;
        this.title = title;
        this.holiday = holiday;
    }
}

File: PersianCalendar/src/main/java/com/byagowi/persiancalendar/entities/IslamicCalendarEvent.kt
package com.byagowi.persiancalendar.entities

import io.github.persiancalendar.calendar.IslamicDate

class IslamicCalendarEvent(date: IslamicDate, title: String, holiday: Boolean) : AbstractEvent<IslamicDate>() {
    init {
        this.date = date
        this.title = title
        this.isHoliday = holiday
    }
}
